src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/OrchestrationConstants.java

```diff
@@ -18,4 +20,12 @@
 
     public static final String YES_VALUE = "Yes";
     public static final String NO_VALUE = "No";
+
+    public static final String APPLICANT = "Applicant";
+    public static final String RESPONDENT = "Respondent";
+
+    // Document Generation
+    public static final String DOCUMENT_FILENAME = "document_filename";
```

Comment:
why don't you use this:

```@JsonIgnoreProperties(ignoreUnknown = true)
@Data
public class CaseDocument {
    @JsonProperty("document_url")
    private String documentUrl;
    @JsonProperty("document_filename")
    private String documentFilename;
    @JsonProperty("document_binary_url")
    private String documentBinaryUrl;
}```

uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd.CaseDocument